src/main/java/org/ibp/api/java/impl/middleware/dataset/DatasetExcelExportServiceImpl.java

```diff
@@ -76,8 +86,11 @@ public List<MeasurementVariable> getColumns(final int studyId, final int dataset
 	public Map<Integer, List<ObservationUnitRow>> getObservationUnitRowMap(final Study study, final DatasetDTO dataset,
 		final Map<Integer, StudyInstance> selectedDatasetInstancesMap) {
 		final Map<Integer, List<ObservationUnitRow>> observationUnitRowMap = new HashMap<>();
+		final List<MeasurementVariableDto> environmentDetails = new ArrayList<>();
```

Comment:
You can move environmentDetauls and environmentConditions inside updateSearchDTOForSummaryData.